geolocator_windows/example/pubspec.yaml

```
@@ -0,0 +1,52 @@
+name: geolocator_windows_example
+description: Demonstrates how to use the geolocator_windows plugin.
+
+# The following line prevents the package from being accidentally published to
+# pub.dev using `pub publish`. This is preferred for private packages.
+publish_to: 'none' # Remove this line if you wish to publish to pub.dev
+
+environment:
+  sdk: ">=2.12.0 <3.0.0"
```

As we are in the progress updating the requirements of the other packages at the moment we might as well use latest settings here:

```suggestion
  sdk: ">=2.15.0 <3.0.0"
```